server/server.go

```diff
@@ -1420,6 +1425,7 @@ func (s *Server) v1DataGet(w http.ResponseWriter, r *http.Request) {
 			result.Explanation, err = types.NewTraceV1(*buf, pretty)
 			if err != nil {
 				writer.ErrorAuto(w, err)
+				return
```

Comment:
From the general code layout, I don't believe this one was omitted with intent.

Reply:
good catch